common/src/sprocket-common/celery/celery.service.ts

```diff
@@ -21,9 +21,19 @@ export class CeleryService {
     private progressChannel: Channel;
 
     async onApplicationBootstrap(): Promise<void> {
+        this.celeryClient.conf.CELERY_BACKEND_OPTIONS = {
+            tls: {
+                servername: "redis.dev.spr.ocket.cloud",
```

Comment:
This value should come from config

Reply:
🦕

Reply:
https://github.com/SprocketBot/sprocket/pull/71/commits/717772cf236a7e75e92a4bbfa401d21b3b8a5afb